api/api.go

```diff
@@ -113,12 +113,12 @@ func (s *Server) Run() {
 	if s.SSL {
 		proto = "https"
 	}
-	log.Info("API Listening on: %v://%s/", proto, s.Addr)
+	log.Infof("API Listening on: %v://%s/", proto, s.Addr)
```

Comment:
Most of these changes shouldn't actually be needed, as the logrus `log.Info` (and friends) functions support printf-style parameters.

Reply:
I haven't tested but when I was reading the code in https://github.com/sirupsen/logrus/blob/master/entry.go#L153-L157 it looks to just pass all the args through to `fmt.Sprintf`.

A side note is that it means you *must* use printf syntax for any log entries that include variables.  If you use string concatenation to build the log message and one of your strings contains a sequence that `fmt` interprets as a placeholder, then it will blow up `fmt`.

Reply:
ugh. if that's true, i'm not a big fan. i've run into issues in the past where i was trying to contain a string that "contained %keywords like %so" and it blew up in my face because it was trying to parse as printf string.
that said, i can live with it.

Reply:
The regular log.Info sends it to fmt.Sprint, whereas the log.Infof sends it to fmt.Sprintf. So, someone could still use log.Info and include strings that `fmt` would consider placeholder without any issues. The actual desired log string itself would just need to be structured slightly differently.
Instead of `log.Infof("My message with some variable %v", variable)`
They could use `log.Info("My message with some variable: ", variable, " and a placeholder %v that will not be changed by fmt")`

Reply:
ok so if Info calls fmt.Sprint and Infof calls fmt.Sprintf that's perfect and what i would expect. and then it seems you're right in making changes like this one.

Reply:
aha that makes sense, missed the `Sprint` vs `Sprintf` 👍